src/lib/snark_worker/rpc_get_work.ml

```diff
@@ -0,0 +1,54 @@
+(* OPAM libraries *)
+open Async
+open Core
+
+(* Local libraries *)
+open Signature_lib
+module Work = Snark_work_lib
+
+(* For versioning of the types here, see:
```

Comment:
Note that you can use odoc convention for links: https://github.com/ocaml/odoc/blob/master/doc/examples/odoc_examples.ml#L4.